addons/account/migrations/8.0.1.1/pre-migration.py

```diff
@@ -35,6 +59,9 @@ def migrate(cr, version):
             cr, 'account', 'exp', 'account.analytic.journal', res[0], True)
     openupgrade.rename_columns(cr, column_renames)
     openupgrade.rename_tables(cr, tables_renames)
+
```

Comment:
Please don't add these extra lines. PEP8 advises not to abuse of these lines and in this case, it doesn't add any extra value.